client/tmclient/client.py

```diff
@@ -64,6 +65,8 @@ def set_on_recv(self, handler):
         await self.connection.send('REFRESH')
 
     async def register(self, username, password):
+        # hash on client side to avoid plaintext passwords
+        password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Comment:
since the server side hashes passwords, wouldn't the double hashing break password checking for login?

Reply:
eh login still works, it's just that the user's password is more of a seed for a randomly generated client password, which is kind of silly